test/processPicker.test.ts

```diff
@@ -0,0 +1,216 @@
+import { should } from 'chai';
+import { RemoteAttachPicker } from '../src/features/processPicker';
+
+suite("Remote Process Picker: Validate quoting arguments.", () => {
```

Comment:
Nice! 
Would be great if you can add one or two  tests with `/` in path. Say `C:/test/mypipe.exe` and `/usr/bin/pipe`